a call of argument 'done'  in a method 'up', migrator.add

zhe1ka opened the pull request:
by any reasons sometimes a callback 'done' may be not called and so our migration will work incorrectly, but a server console doesn't have any message about this problem. I suggest to show some message that has a description what was done incorrectly.

zhe1ka committed:
by any reasons sometimes a callback 'done' may be not called
 and so our migration will work incorrectly, but a server console
 doesn't have any message about this problem. I suggest to show
 some message that has a description what was done incorrectly.

emirotin commented:
You've changed the generated JS file. Source files are in CoffeeScript, please update your PR by changing the source file and then regenerating the JS file using the build script https://github.com/emirotin/mongodb-migrations/blob/develop/package.json#L7

zhe1ka committed:
by any reasons sometimes a callback 'done' may be not called
 and so our migration will work incorrectly, but a server console
 doesn't have any message about this problem. I suggest to show
 some message that has a description what was done incorrectly.

emirotin commented:
After having the second look I don't think this will work as expected.
The `checkCallDone` is se to `true` in the callback which may (and in the majority of cases _will_ be called asynchronously). But then this variable is synchronously checked, so it will always be `false` and the warning will be printed.

In addition to that there's already the migration timeout feature that will reject your migration with the `new Error("migration timed-out");` error, including the case when you just never call the callback. It's available since v0.8.5.

I'm going to close this PR but please correct me if I'm wrong with my understanding of the behavior in the first paragraph.
